mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/query/nativequery/MongoDBNativeQueryPaginationTest.java

```diff
@@ -25,21 +26,21 @@
  */
 public class MongoDBNativeQueryPaginationTest extends OgmTestCase {
 
-	public static final String QUERY_FIND = "db.WILDE_POEM.find( { 'author' : 'Oscar Wilde' } )";
-	public static final String QUERY_AGGREGATE_MATCH = "db.WILDE_POEM.aggregate( [ { '$match' : { 'author' : 'Oscar Wilde' } } ] )";
+	public static final String QUERY_FIND = "db.WILDE_POEM.find( { '$query' : { 'author' : 'Oscar Wilde' }, '$orderby' : { 'name' : 1 } } )";
+	public static final String QUERY_AGGREGATE_MATCH = "db.WILDE_POEM.aggregate( [ { '$match' : { 'author' : 'Oscar Wilde' } }, { '$sort': {'name': 1 } } ] )";
 
 	private final OscarWildePoem portia = new OscarWildePoem( 1L, "Portia", "Oscar Wilde", 1881 );
 	private final OscarWildePoem athanasia = new OscarWildePoem( 2L, "Athanasia", "Oscar Wilde", 1879 );
 	private final OscarWildePoem imperatrix = new OscarWildePoem( 3L, "Ave Imperatrix", "Oscar Wilde", 1882 );
 	private final OscarWildePoem intellectualis = new OscarWildePoem( 4L, "Amor Intellectualis", "Oscar Wilde", 1881 );
-	private final OscarWildePoem apologia = new OscarWildePoem( 5L, "Apologias ", "Oscar Wilde", 1881 );
+	private final OscarWildePoem apologia = new OscarWildePoem( 5L, "Apologias", "Oscar Wilde", 1881 );
 	private final OscarWildePoem easter = new OscarWildePoem( 6L, "Easter Day", "Oscar Wilde", 1881 );
 	private final OscarWildePoem rome = new OscarWildePoem( 7L, "Rome Unvisited", "Oscar Wilde", 1881 );
 	private final OscarWildePoem miniato = new OscarWildePoem( 8L, "San Miniato", "Oscar Wilde", 1881 );
 	private final OscarWildePoem liberty = new OscarWildePoem( 9L, "Sonnet to Liberty", "Oscar Wilde", 1881 );
 	private final OscarWildePoem vita = new OscarWildePoem( 10L, "Vita Nuova", "Oscar Wilde", 1881 );
 
-	private final OscarWildePoem[] poems = { portia, athanasia, imperatrix, intellectualis, apologia, easter, rome, miniato, liberty, vita };
+	private final OscarWildePoem[] poems = { intellectualis, apologia, athanasia, imperatrix, easter, portia, rome, miniato, liberty, vita };
```

Comment:
`intellectualis` before `apologias`?

Reply:
yes the name of the poem intellectualis is "Amor Intellectualis"